pkg/index/job/correction/usecase/corrector.go

```diff
@@ -51,23 +51,19 @@ func New(cfg *config.Data) (r runner.Runner, err error) {
 
 	eg := errgroup.Get()
 
-	cOpts, err := cfg.Corrector.Discoverer.Client.Opts()
+	dOpts, err := cfg.Corrector.Discoverer.Client.Opts()
```

Comment:
> **NOTE**
There are `cOpts` and `dOpts` in previous implementations, but since `cOpts` are not used elsewhere in this case, only `dOpts` seem to be good.